test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Json.Tests.ps1

```diff
@@ -40,6 +40,10 @@ Describe 'ConvertTo-Json' -tags "CI" {
         $ps.Dispose()
     }
 
+    It "Should accept minimum depth as 0." {
+        { $output = 1 | ConvertTo-Json -Depth 0 } | Should -Not -Throw
```

Comment:
This test needs to validate the expected output

Reply:
@SteveL-MSFT Added. Thanks